localstack.yaml

```diff
@@ -0,0 +1,177 @@
+package:
+  name: localstack
+  version: 4.0.3
+  epoch: 0
+  description: A fully functional local AWS cloud stack. Develop and test your cloud & Serverless apps offline
+  copyright:
+    - license: Apache-2.0
+    - license: LicenseRef-LICENSE
+      license-path: LICENSE.txt
+    - license: LicenseRef-EULA
+      license-path: docs/end_user_license_agreement/README.md
+  dependencies:
+    runtime:
+      - binutils
+      - ca-certificates-bundle
+      - corepack
+      - coreutils
+      - curl
+      - git
+      - gnupg
+      - gnutar
+      - groff
+      - iproute2
+      - iputils
+      - libatomic
+      - libexpat1
+      - make
+      - npm
+      - nss-passwords
+      - openjdk-11-default-jvm
+      - openssl
+      - pixz
+      - procps
+      - py${{vars.py-version}}-tzdata
+      - python-${{vars.py-version}}
+      - unzip
+      - xz
+      - zip
+
+vars:
+  py-version: 3.11
+
+environment:
+  contents:
+    packages:
+      - bash
+      - build-base
+      - busybox
+      - ca-certificates-bundle
+      - git
+      - openssf-compiler-options
+      - py${{vars.py-version}}-build
+      - py${{vars.py-version}}-pip
+      - py${{vars.py-version}}-pytest
+      - py${{vars.py-version}}-tz
+      - python-${{vars.py-version}}
+      - tzdata
+      - uv
+
+pipeline:
+  - uses: git-checkout
+    with:
+      repository: https://github.com/localstack/localstack
+      tag: v${{package.version}}
+      expected-commit: aa795ed1c04a8285c01c96aa7ab79c17489f5766
+
+  - runs: |
+      mkdir -p ${{targets.destdir}}/usr/share/localstack
+      mkdir -p ${{targets.destdir}}/tmp/localstack
+
+  - name: Build
+    runs: |
+      make install-runtime
```

Comment:
Seems like it is coming from here https://github.com/localstack/localstack/blob/bdfdf1509eb140772e5ed58972ad59e716c6fe5d/Makefile#L52 with these being runtime deps https://github.com/localstack/localstack/blob/master/requirements-runtime.txt

Why are we excluding them outside our venv?

Reply:
these are being executed inside a virtualenv and then we copy whole `.venv` directory to `/usr/share/localstack` later below. 

```yaml
mv .venv ${{targets.destdir}}/usr/share/localstack/
```

Reply:
This is built before it's then installed into vent. It has to be done here.